<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserTuneOptions</class>
 <widget class="QWidget" name="UserTuneOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>User Tune</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="chb_showIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Show music icon in roster</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="chb_allowSendMusicInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Allow to send information about playing music</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chb_dontSendURI">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Never send a URL tag</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="l_tags">
     <property name="text">
      <string>Show format:</string>
     </property>
     <property name="buddy">
      <cstring>le_format</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_format">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;%A - artis&lt;br/&gt;%L - length&lt;br/&gt;%R - rating&lt;br/&gt;%S - source&lt;br/&gt;%T - title&lt;br/&gt;%N - track&lt;br/&gt;%U - URL&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="l_playerName">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Player name:</string>
     </property>
     <property name="buddy">
      <cstring>cb_playerName</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cb_playerName"/>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btn_refreshPlayers">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="l_mpris_version">
     <property name="text">
      <string>MPRIS Version:</string>
     </property>
     <property name="buddy">
      <cstring>cb_mpris_version</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cb_mpris_version"/>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>210</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>btn_refreshPlayers</zorder>
  <zorder>cb_playerName</zorder>
  <zorder>l_playerName</zorder>
  <zorder>l_tags</zorder>
  <zorder>le_format</zorder>
  <zorder>cb_mpris_version</zorder>
  <zorder>l_mpris_version</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>chb_showIcon</zorder>
  <zorder>chb_allowSendMusicInfo</zorder>
 </widget>
 <tabstops>
  <tabstop>le_format</tabstop>
  <tabstop>cb_playerName</tabstop>
  <tabstop>btn_refreshPlayers</tabstop>
  <tabstop>cb_mpris_version</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chb_allowSendMusicInfo</sender>
   <signal>toggled(bool)</signal>
   <receiver>chb_dontSendURI</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modified()</slot>
 </slots>
</ui>
